bythepixel/kyrio-go2broadband
# Fix IE Support

Closes #8 
Closes #9 
#### How should the reviewer test this?
- Navigate to http://dev.bythepixel.com/kyrio
- Ensure full browser support
